<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenVis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../img/img.qrc">
    <normaloff>:/resources/img/logos/genvis.svg</normaloff>:/resources/img/logos/genvis.svg</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_widget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_recent_files">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <property name="icon">
      <iconset resource="../../img/img.qrc">
       <normaloff>:/resources/img/icons/open_recent.svg</normaloff>:/resources/img/icons/open_recent.svg</iconset>
     </property>
    </widget>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="menu_recent_files"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="action_save_all"/>
    <addaction name="separator"/>
    <addaction name="action_print"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_delete"/>
    <addaction name="action_clear_all"/>
   </widget>
   <widget class="QMenu" name="menu_options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_settings"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_about_genvis"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_options"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="standard_toolbar">
   <property name="windowTitle">
    <string>Standard Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_print"/>
  </widget>
  <widget class="QToolBar" name="tools_toolbar">
   <property name="windowTitle">
    <string>Tools Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="applications_toolbar">
   <property name="windowTitle">
    <string>Applications Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_exec_gensec"/>
   <addaction name="action_exec_genrep"/>
  </widget>
  <action name="action_new">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/new.svg</normaloff>:/resources/img/icons/new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Create a new file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/open.svg</normaloff>:/resources/img/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open an existing file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/save.svg</normaloff>:/resources/img/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="statusTip">
    <string>Save the current document on disk.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/save_as.svg</normaloff>:/resources/img/icons/save_as.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="statusTip">
    <string>Save the current document under a new name.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_save_all">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/save_all.svg</normaloff>:/resources/img/icons/save_all.svg</iconset>
   </property>
   <property name="text">
    <string>Save A&amp;ll</string>
   </property>
   <property name="statusTip">
    <string>Save all documents on disk.</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/print.svg</normaloff>:/resources/img/icons/print.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="statusTip">
    <string>Print the current table.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/cut.svg</normaloff>:/resources/img/icons/cut.svg</iconset>
   </property>
   <property name="text">
    <string>C&amp;ut</string>
   </property>
   <property name="statusTip">
    <string>Cut the current selection content to the clipboard.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/copy.svg</normaloff>:/resources/img/icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the current selection content to the clipboard.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/paste.svg</normaloff>:/resources/img/icons/paste.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste the clipboard content into the current selection.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/help.svg</normaloff>:/resources/img/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/about.svg</normaloff>:/resources/img/icons/about.svg</iconset>
   </property>
   <property name="text">
    <string>About &amp;Sequence-ToolKit</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/delete.svg</normaloff>:/resources/img/icons/delete.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="statusTip">
    <string>Remove the current selection.</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/quit.svg</normaloff>:/resources/img/icons/quit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Close all tabs and quit the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/close.svg</normaloff>:/resources/img/icons/close.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="statusTip">
    <string>Close the current tab.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_clear_all">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/delete_all.svg</normaloff>:/resources/img/icons/delete_all.svg</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear All</string>
   </property>
   <property name="statusTip">
    <string>Delete all the content of the table.</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="action_about_genvis">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/about_variant.svg</normaloff>:/resources/img/icons/about_variant.svg</iconset>
   </property>
   <property name="text">
    <string>About &amp;GenVis</string>
   </property>
   <property name="toolTip">
    <string>About GenVis</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/settings.svg</normaloff>:/resources/img/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_exec_genrep">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/logos/genrep.svg</normaloff>:/resources/img/logos/genrep.svg</iconset>
   </property>
   <property name="text">
    <string>GenRep</string>
   </property>
   <property name="toolTip">
    <string>Open Report Generator</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/undo.svg</normaloff>:/resources/img/icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/icons/redo.svg</normaloff>:/resources/img/icons/redo.svg</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_exec_gensec">
   <property name="icon">
    <iconset resource="../../img/img.qrc">
     <normaloff>:/resources/img/logos/gensec.svg</normaloff>:/resources/img/logos/gensec.svg</iconset>
   </property>
   <property name="text">
    <string>GenSec</string>
   </property>
   <property name="toolTip">
    <string>Open Sequence Generator</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../img/img.qrc"/>
 </resources>
 <connections/>
</ui>
